src/spatial_grid.jl

```diff
@@ -1,35 +1,21 @@
 """
-` Rasterize a points cloud in 2D`
+    rasterize_points(cloud::PointCloud, dx::AbstractFloat)
+    rasterize_points(cloud::Matrix{AbstractFloat}, dx::AbstractFloat)
```

Comment:
The fact that you need to define this makes me worried that our `PointCloud` abstraction isn't really right.  Is it that you have a position-only point cloud and you happen to have the positions stored in a matrix?

Side note - `cloud::Matrix{AbstractFloat}` could be more precisely written as `cloud::Matrix{<:AbstractFloat}`.  IIRC Jeff or someone at juliacon may have said this would be real syntax soon (@andyferris - do you remember?)